src/solvers/dg_p4est/dg_2d.jl

```diff
@@ -0,0 +1,154 @@
+function rhs!(du, u, t,
```

Comment:
What is different here in comparison to the TreeMesh version?

Reply:
There's no `prolong2boundaries` and no mortar stuff.